src/callback.rs

```diff
@@ -499,7 +499,7 @@ pub fn pointer_scroll(callback: extern "C" fn(view: WlcView, time: u32,
 /// use rustwlc::Point;
 /// use rustwlc::input::pointer;
 ///
-/// extern fn pointer_motion(view: WlcView, time: u32, point: &Point) -> bool {
+/// extern fn pointer_motion(view: WlcView, time: u32, point: Point) -> bool {
```

Comment:
I made this change because cargo test failed. (see commit message: I copied the output)

I tried to run the unit tests as part of travis, but gave up after seeing that ubuntu 12.04 doesn't have the wlc package.